docs/k8s-dashboard/access-k8s-dashboard.md

```diff
@@ -0,0 +1,35 @@
+# Access K8S Dashboard
+
+This doc is for ops/admin users. If ops/admin deployed OpenPAI with k8s RBAC enabled. The k8s dashboard with `https` enabled  will be deployed. And authentication is needed to access k8s-dashboard.
+
+Currently, we only support access k8s dashboard through service token.
+
+***Please make sure your OpenPAI cluster is enabled HTTPS. Otherwise you need to access dashboard through kube-proxy (refer to [access dashboard](https://github.com/kubernetes/dashboard/blob/master/docs/user/accessing-dashboard/1.7.x-and-above.md))***
+
```

Comment:
It seems there should be a prerequisite section: in order to run kubectl, user need to set up kubeconfig correctly. Please also give a reference to how to have a kubeconfig file.

Reply:
Add prerequisite section. For how to enable https for PAI. Currently, we don't have a doc, and https is coupled with AAD, refer to issue: #4108